src/codeu/scripts/history.sql

```diff
@@ -0,0 +1,57 @@
+4.14
```

Comment:
Does this file remain mostly the same? If so, then can you put a comment at the top for readers? If that's not possible / not how it's generated then that's OK. But if it _is_ possible, please do so :)